app/src/main/java/com/ggaebiz/ggaebiz/presentation/ui/setting/SettingViewModel.kt

```diff
@@ -0,0 +1,69 @@
+package com.ggaebiz.ggaebiz.presentation.ui.setting
+
+import com.ggaebiz.ggaebiz.domain.usecase.GetCharacterIdxUseCase
+import com.ggaebiz.ggaebiz.domain.usecase.SetTimerSettingUseCase
+import com.ggaebiz.ggaebiz.presentation.common.base.BaseViewModel
+
+data class SettingState(
+    val selectedCharacterIdx: Int = 0,
+    val level: Int = 1,
+    val hour: Int = 0,
+    val minute: Int = 30,
+    val buttonEnabled: Boolean = true
+)
```

Comment:
```suggestion
data class SettingState(
    val selectedCharacterIdx: Int = 0,
    val level: Int = 1,
    val hour: Int = 0,
    val minute: Int = 30,
){
    val buttonEnabled = ( hour != 0 || minute != 0)
}
```

제안드립니다~ ( 제안이라 스킵해도 괜찮! )